src/main/java/redis/clients/jedis/search/aggr/Params.java

```diff
@@ -0,0 +1,20 @@
+package redis.clients.jedis.search.aggr;
+
+public class Params {
```

Comment:
We can find other alternatives of adding this new class.

Reply:
Could a simple Map be a good alternative?

Reply:
Thanks for the suggestion. Replaced with Map<String, Object>